patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -219,15 +219,27 @@
              this.func_72964_e(i, j).func_76612_a(p_72838_1_);
              this.field_72996_f.add(p_72838_1_);
              this.func_72923_a(p_72838_1_);
-@@ -1268,6 +1332,7 @@
+@@ -1227,6 +1291,7 @@
+         IBlockState iblockstate = Blocks.field_150348_b.func_176223_P();
+         BlockPos.PooledMutableBlockPos blockpos$pooledmutableblockpos = BlockPos.PooledMutableBlockPos.func_185346_s();
+ 
++        if (p_191504_3_ && !net.minecraftforge.event.ForgeEventFactory.gatherCollisionBoxes(this, p_191504_1_, p_191504_2_, p_191504_4_)) return true;
+         try
+         {
+             for (int k1 = i; k1 < j; ++k1)
+@@ -1268,9 +1333,11 @@
                                  }
  
                                  iblockstate1.func_185908_a(this, blockpos$pooledmutableblockpos, p_191504_2_, p_191504_4_, p_191504_1_, false);
 +                                net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(new net.minecraftforge.event.world.GetCollisionBoxesEvent(this, null, p_191504_2_, p_191504_4_));
```

Comment:
Could you change these existing calls to the new method too?

Reply:
done